runtime/src/main/jni/NetworkDomainCallbackHandlers.h

```diff
@@ -0,0 +1,278 @@
+//
+// Created by pkanev on 3/8/2017.
+//
+
+#ifndef NETWORKDOMAINCALLBACKHANDLERS_H
+#define NETWORKDOMAINCALLBACKHANDLERS_H
+
+
+#include <include/v8.h>
+#include <v8_inspector/src/inspector/v8-network-agent-impl.h>
+#include "JsV8InspectorClient.h"
+#include "ArgConverter.h"
+
+namespace tns {
+namespace NetworkDomainCallbackHandlers {
+
+static const char* FrameId = "NSFrameIdentifier";
+static const char* LoaderId = "NSLoaderIdentifier";
+
+static void ResponseReceivedCallback(const v8::FunctionCallbackInfo<v8::Value>& args) {
+    try {
+        auto networkAgentInstance = V8NetworkAgentImpl::Instance;
+
+        if (!networkAgentInstance) {
+            return;
+        }
+
+        if (args.Length() != 0 && !args[0]->IsObject()) {
+            return;
+        }
+
+        auto isolate = args.GetIsolate();
+
+        v8::HandleScope scope(isolate);
+
+        auto context = isolate->GetCurrentContext();
+
+        v8::Local<v8::Object> argsObj = args[0]->ToObject();
+
+        if ((!argsObj->Has(context, ArgConverter::ConvertToV8String(isolate, "requestId")).FromMaybe(false) ||
```

Comment:
why not use a for statement - we check for 4 different parameters in the same pattern